public/content/documentation/web/component/link.md

```diff
@@ -6,75 +6,485 @@ How to test a link
 
 ### #a11y - Web Accessibility Acceptance Criteria
 
-How to test a header
+How to test a link
 
 1. Test keyboard only, then screen reader + keyboard actions
 
-   - Skip-links: Focus moves directly to the header or navigation
-
-   - Tab: Nothing, headings are not focusable unless they are actionable
-
-   - Arrow-keys: Headings are browsed
+   - Tab: Focus visibly moves to the link.
+   - Enter: Activates the link.
 
 2. Test mobile screenreader gestures
 
-   - Swipe: Focus moves directly to the header or navigation
-
-   - Doubletap: This typically activates most elements
+   - Swipe: Focus moves to the link
```

Comment:
Up to you for consistency, either:
a. add periods to Swipe/Doubletap (this step), Name/Role (below step) OR
b. remove periods from Tab/Enter (above step)